nepomuk/src/main/java/edu/asu/diging/gilesecosystem/nepomuk/core/files/impl/FilesManager.java

```diff
@@ -86,5 +86,14 @@ public String getRelativePathOfFile(IFile file) {
     public List<String> getKnownUsernames() {
         return databaseClient.getUsernames();
     }
-
+    
+    @Override
+    public void deleteFile(String fileId) {
+        databaseClient.deleteFile(fileId);
+    }
+    
+    @Override
+    public List<IFile> getFilesByDocumentId(String uploadId) {
```

Comment:
let's move that method up under `getFilesByUploadId` since they are very similar.